package-lock.json

```diff
@@ -10,7 +10,7 @@
       "integrity": "sha1-hiRnWMfdbSGmR0/whKR0DsBesh8=",
       "dev": true,
       "requires": {
-        "mime-types": "2.1.16",
```

Comment:
splits the deps auto-updates out into a separate commit as it's otherwise unrelated to this PR